nesta/packages/quality_check/quality_calculations.py

```diff
@@ -0,0 +1,68 @@
+import pandas as pd
+import itertools
+
+def missing_values(data_frame):
+
+    """
+
+    """
+    return data_frame.isnull().sum().sort_values(ascending=False)
+
+def missing_value_column_count(data_frame):
+
+    bins_list = [i for i in range(100+1) if i%10 == 0]
+    out = pd.cut(data_frame.isnull().mean().apply(lambda x: round(100*x,5)), bins=bins_list, include_lowest=True)
+
+    return out.value_counts(sort=False)
+
+def missing_value_count_pair_both(data_frame):
+#include here or in script itself
+    pair_list = []
+    for pair in itertools.combinations_with_replacement(data_frame.columns,2):

```

Comment:
This first loop and the list of pairs is redundant. Just add the pairs as keys to the null_dict as they are generated.